_pages/languages.md

```diff
@@ -92,27 +136,22 @@ TTS engines are made available by different vendors, and they each have differen
 * [List of languages supported by MaryTTS](http://mary.dfki.de/)
 * [List of languages supported by BingTTS](https://docs.microsoft.com/en-us/azure/cognitive-services/speech/api-reference-rest/s)
 
-Once you've found an TTS engine for your language, you will need to configure Mycroft to use it. This is done through the [mycroft.conf](https://mycroft.ai/documentation/mycroft-conf/) file. 
+Once you've found an TTS engine for your language, you will need to configure Mycroft to use it. This is done through the [mycroft.conf](https://mycroft.ai/documentation/mycroft-conf/) file.
 
 #### Editing `mycroft.conf` to specify a different language for TTS
 
-Locate your `mycroft.conf` file, and open it for editing on the Linux command line, using a tool like `vi` or `nano`. 
+Locate your `mycroft.conf` file, and open it for editing on the Linux command line, using a tool like `vi` or `nano`.
 
 ```json
 {
   “tts”: {
-  “module”: “google_cloud”,    <- this should be set to the name of your TTS provider (ie "google_cloud", "wit" etc)
-  “google_cloud”: {            <- this should be set to the name of your TTS provider (ie "google_cloud", "wit" etc)
-    “lang”: “hi-in”,           <- the IETF BCP-47 language code for your language (shown is Hindi as spoken in India)
-    “credential”: {            <- some STT engines require credentials - check the documentation for the STT engine
-      “json”: {
-      }
-    }
+  “module”: “google”,
+  “lang": "hi"
```

Comment:
This looks incorrect: I think It should be

```json
  "tts": {
    "module": "google",
    "google": {
      "lang": "hi"
    }
  }
```

And maybe there could be an indication that there are several options other than google?

Reply:
yah that looks right